packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -76,54 +102,7 @@ export class StatsAnalyzer extends EventsScope {
     this.correlationId = config.correlationId;
     this.mqaSentCount = -1;
     this.lastMqaDataSent = {
-      resolutions: {
-        video: {send: {}, recv: {}},
-        audio: {send: {}, recv: {}},
-        share: {send: {}, recv: {}},
-      },
-      video: {send: {}, recv: {}},
-      audio: {send: {}, recv: {}},
-      share: {send: {}, recv: {}},
-    };
-    this.localMQEStats = {
-      audio: {
-        RX: {
-          packetsLost: [],
-          jitter: [],
-          latency: [],
-          bitRate: [],
-        },
-        TX: {
-          packetsLost: [],
-          jitter: [],
-          latency: [],
-          bitRate: [],
-        },
-      },
-      video: {
-        RX: {
-          packetsLost: [],
-          jitter: [],
-          latency: [],
-          bitRate: [],
-          frameRate: [],
-          resolutionWidth: [],
-          resolutionHeight: [],
-          requestedKeyFrame: [],
-          receivedKeyFrame: [],
-        },
-        TX: {
-          packetsLost: [],
-          jitter: [],
-          latency: [],
-          bitRate: [],
-          frameRate: [],
-          resolutionWidth: [],
-          resolutionHeight: [],
-          requestedKeyFrame: [],
-          receivedKeyFrame: [],
-        },
-      },
+      resolutions: {},
```

Comment:
i feel that some of these needs to be defined , else when testing it out you might see errors seeing recv or undefined 

can you test it and confirm

Reply:
We generate the previous mqa items dynamically, so instead of `this.lastMqaDataSent['audio']`  we have `this.lastMqaDataSent['audio-recv-1']`, `this.lastMqaDataSent['video-send-2']` etc.  I have tested it and it is able to handle the comparisons between last and current.